web/src/common/constants.js

```diff
@@ -0,0 +1 @@
+export const SECTION_PADDING = "50px 0 50px 0";
```

Comment:
can we make this increments of 8 please. So either 48px or 60px

Reply:
okie